### PR TITLE
Ryanyae 127 gi testimony

### PR DESCRIPTION
**Pull request by ryanyae**
## Description

Made testimony cards for project gallery.

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by ryanyae** [comment]
[implement testimony for project get involve page](https://trello.com/c/lRxx4TW2/127-implement-testimony-for-project-get-involve-page)

**Comment by ElisaHu1** [comment]
minor: are you able to change this to 04 to match the figma? 
image

**Comment by ryanyae** [comment]
> minor: are you able to change this to 04 to match the figma? image

@ElisaHu1 Slipped my mind, all fixed!

File: src/pages/Project/types.tsx
Comment (ElisaHu1) [inline src/pages/Project/types.tsx:1]: like your design of combing the cardholder for both the description and testimony

File: src/components/ProjectGallery/ProjectGallery.tsx [hunk 1]
```diff
@@ -67,16 +79,16 @@ const ProjectGallery = (props:PropsOBJ) => {
...
-                                    <div className='box-content' key={index}>
-                                        <div className='image-container'>
-                                            <img src={obj.img} alt='' className='gallery-img' />
-                                        </div>
-                                        <h1 className='gallery-title'>{obj.title}</h1>
-                                        <p className='gallery-description'>L{obj.description}</p>
-                                    </div>
-                                ))
+                                    itemArray.map((obj, index) => {
+                                    switch (obj.cardType) {
```
Comment (ElisaHu1) [inline src/components/ProjectGallery/ProjectGallery.tsx:83]: good work!

**Review by ElisaHu1 — APPROVED** [review]
clear code! great job

**Pull request merged**